commands/types.go

```diff
@@ -2,7 +2,33 @@ package commands
 
 // define all the types used in a common file
 
+// Project defines the project response object
 type Project struct {
-	Title string `json:"title"`
-	Guid  string `json:"guid"`
+	AuthorEmail    string      `json:"authorEmail"`
```

Comment:
really handy tool for creating these structs from existing json

https://github.com/ChimeraCoder/gojson